openmls/src/treesync/node/leaf_node/capabilities.rs

```diff
@@ -152,7 +152,7 @@ impl Capabilities {
     }
 
     /// Check if these [`Capabilities`] contain all the extensions.
-    pub(crate) fn contain_extensions(&self, extension: &Extensions) -> bool {
+    pub(crate) fn are_extensions_supported(&self, extension: &Extensions) -> bool {
```

Comment:
I have renamed this method because I believe the previous naming leads to confusion. It does not check if the capabilities contain all the extensions but rather if the extensions are in the capabilities.